packages/ember/src/components/icon.gts

```diff
@@ -0,0 +1,50 @@
+import Component from '@glimmer/component';
+import type { NrgIconValue } from '../';
+
+declare type SubtleColorType =
+  | 'primary-subtle'
+  | 'secondary-subtle'
+  | 'success-subtle'
+  | 'danger-subtle'
+  | 'warning-subtle'
+  | 'info-subtle'
+  | 'light-subtle'
+  | 'dark-subtle';
+
+declare type ColorType =
+  | 'primary'
+  | 'secondary'
+  | 'success'
+  | 'danger'
+  | 'warning'
+  | 'info'
+  | 'light'
+  | 'dark';
+
+interface IconSignature {
+  Element: HTMLDivElement;
+  Args: {
+    type: NrgIconValue;
+    backgroundColor?: SubtleColorType | ColorType;
+    color: ColorType;
+    circular?: boolean;
+  };
+}
+
+export default class Icon extends Component<IconSignature> {
+  get classList() {
+    let classes = ['d-flex', 'justify-content-center', 'fs-2', 'm-2'];
+    if (this.args.circular) {
+      classes.push('rounded-circle', 'p-3', `bg-${this.args.backgroundColor}`);
```

Comment:
The styling of the icon with or without the circular param is technically the same, minus the padding added for the circle itself. Either way, this would be good for if they want a color like primary in the background but the icon to be white or something like that! Updating it now